ArduCopter/inertia.cpp

```diff
@@ -17,6 +17,13 @@ void Copter::read_inertia()
     current_loc.lng = loc.lng;
     current_loc.lat = loc.lat;
 
+    // Get XYZ position and velocity in NEU and cm
```

Comment:
No, actually the NEU and cm part. It gets it in NED and m and then does the conversions.